portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,5 +29,26 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello Sean Tisdale!</h1>");
+
+    
+    String json = messages();
+
+   response.setContentType("text/html");
+    response.getWriter().println(json);
```

Comment:
This will return:

```
<h1> Hello Sean Tisdale</h1>
This is my first time using Json```

However, the instructions say that this should return the entire ArrayList as a  JSON string:

```Modify DataServlet so its doGet() function returns the ArrayList as a JSON string. Confirm this works by running a dev server and navigating to the /data URL in your browser.
```

So the resulting output should be the following (with the ArrayList formatted using GSON, not by hand ;) ):

```['This is my first time using Json', 'I wonder if a guy named Jason made Json', 'I love pie']```

Can you modify this exercise, and confirm this is what you see when you go to the browser?

Reply:
Yes the hello message and my custom message both showed

Reply:
I'm a little confused right now on using Gson so this may take awhile. Also is that second example supposed to be blank?

Reply:
actually I think I may have solved it. Pushing soon

Reply:
Sorry! Updated.